style(*): grammar, spelling, punctuation (#539)
discordjs/guide: guide/miscellaneous/useful-packages.md

You can log to the terminal, you can log to a file, etc.  
"But wait," I hear you cry, "what's wrong with `console.log`?".  

> Well, the answer is simple: `console.log` is slow, very slow, and not very versatile.

Reviewer: ```suggestion
 Well, the answer is simple: `console.log` is slow and not quite versatile.
```

> Whenever you call `console.log`, your program halts; it has to wait for console.log to finish.
> While it does that, your program can do nothing else, which does not sound good.

Reviewer: ```suggestion
Whenever you call `console.log`, your program halts; it cannot do anything until `console.log` finishes, which does not sound good.
```
